.buildkite/test/run.sh

```diff
@@ -41,20 +41,32 @@ function retry {
   return 0
 }
 
-# Only for job 2 = Node-API
-if [ "$BUILDKITE_PARALLEL_JOB" = "2" ]; then
-  export PRISMA_FORCE_NAPI=true
```

Comment:
Nothing necessary for job 2 any more?

Reply:
Nope
Job 0 Tests all with the Node-API Library
Job 1 Tests all with the Binary

Reply:
Nice.